backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/picking/pickingslot/PickingSlotRepoQuery.java

```diff
@@ -71,4 +75,18 @@ private PickingSlotRepoQuery(
 		this.onlyNotClosedOrNotRackSystem = onlyNotClosedOrNotRackSystem != null ? onlyNotClosedOrNotRackSystem : true;
 		this.pickingSlotBarcode = pickingSlotBarcode;
 	}
+
+	@Value
+	@Builder
+	public static class CreatePickingSlotRepoQueryReq
```

Comment:
why not moving this class to PickingSlotViewFactory and make it private?

Reply:
yeah, sure, why not